README.md

```diff
@@ -1,2 +1,44 @@
-# -coc-guidelines-taskforce
-Repository to update the Enforcement Manual and Reporting Guidelines for The Carpentries Code of Conduct.
+# The Carpentries Code of Conduct 
+## Reporting Guidelines and Enforcement Guidelines Task Force Charter
+
+__Context__  
+In the last year, incidents surrounding [The Carpentries Code of Conduct (CoC)](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html) highlighted issues with enforcing the CoC and adjudicating reported CoC violations. As a result, and due to Software and Data Carpentry having merged to form The Carpentries, there have been updates to governance and policy documents, including [updates to the Code of Conduct](https://carpentries.org/blog/2018/09/coc-revision-release/).
+
+Updates to the CoC were segmented in three parts: 1) The Carpentries Code of Conduct, 2) Incident Reporting Guidelines, and 3) Enforcement Manual. The process of updating 1) and 2) took five months (from April 2018 through September 2018)
+
+Since completion of the CoC update, there have been two CoC incidents reported.
+
+* __Incident #1:__ Reported 2018-09-07, Resolved 2018-10-17 
+* __Incident #2:__ Reported 2018-09-27, Resolved 2018-10-19
+
+For more information, see the [Code of Conduct Transparency Report](https://github.com/carpentries/executive-council-info/blob/master/code-of-conduct-transparency-reports/2018-11-09-coc-transparency-report.md).
+
+These two incidents were assessed under the current enforcement manual, which prevented the committee from arriving at a resolution swiftly. The Executive Council has expressed concern with the lack of timeliness as it relates to resolving CoC incident reports. The CoC Committee serves to respond and resolve CoC incident reports, and update documentation. Though work has been completed to update our Code of Conduct, the CoC committee lacks the expertise to update the reporting guidelines and enforcement manual. Such updates include, but are not limited to: 
```

Comment:
'the CoC committee lacks the expertise' -> 'the CoC committee could use additional expertise'